<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataShowSubWidget</class>
 <widget class="QWidget" name="DataShowSubWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget_Comment"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_Struct"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_Enum"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
